packages/sui-react-web-vitals/src/index.js

```diff
@@ -46,7 +46,15 @@ export const DEVICE_TYPES = {
 }
 
 const getNormalizedPathname = pathname => {
-  return pathname.replaceAll('*', '_').replace(/\\/g, '')
+  return pathname.replace(/[^a-z0-9]/gi, '')
+}
+
+const getPathname = route => {
+  return route?.path || route?.regexp?.toString()
+}
+
+const getPathIsRegexp = route => {
```

Comment:
Returns a path or a boolean saying that path is a regexp?, for me it's not clear, maybe could be something like `isPathARegExp` or simply `isRegExp`...

Reply:
good point, @kikoruiz !

perhaps it's more clear this way... what do you think?

 https://github.com/SUI-Components/sui/pull/1640/commits/2bc916d36f4cca5c6692da2233ff85153c023731